bootstrap.py

```diff
@@ -1377,7 +1384,9 @@ def exec_service(service, command, failonerror=True):
             if hostid is not None:
                 disassociate_host(hostid)
                 delete_host(hostid)
-        if 'katello-agent' not in options.skip:
+        if 'katello-agent' in options.skip:
+            print_warning("Skipping the installation of the Katello Agent is now the default behavior. passing --skip katello-agent is deprecated")
+        if 'katello-agent' or 'katello-host-tools' not in options.skip:
```

Comment:
this is always `True`, you want:

```suggestion
        if 'katello-agent' not in options.skip or 'katello-host-tools' not in options.skip:
```

Reply:
:+1: perfecto.